kafka-connect-fitbit-source/src/main/java/org/radarbase/connect/rest/fitbit/user/ServiceUserRepositoryLegacy.java

```diff
@@ -76,6 +80,19 @@ public class ServiceUserRepositoryLegacy implements UserRepository {
  
    public ServiceUserRepositoryLegacy() {
      this.client = new OkHttpClient.Builder()
+          .addInterceptor(chain -> {
```

Comment:
@yatharthranjan I added an interceptor here because the `OAuth2Client` library has been deprecated (was in MP) for the ktor implementations (which is in `ServiceUserRepository`).